docs/src/tutorials/conic/ellipse_fitting.jl

```diff
@@ -0,0 +1,416 @@
+# # Example: least-squares fitting of circles and ellipses
```

Comment:
This needs a license header

Reply:
Which license exactly do you mean?

Reply:
You can copy-paste this:
https://github.com/jump-dev/JuMP.jl/blob/00c3b35d8f0d894e0ff70768b58ee200631cc6a2/docs/src/tutorials/conic/dualization.jl#L1-L5

Reply:
got it, thanks